src/main/java/fr/uge/chargepointconfiguration/chargepoint/ConfigurationServer.java

```diff
@@ -58,7 +58,10 @@ public void onMessage(WebSocket conn, String message) {
             + ": "
             + message);
     var webSocketMessage = WebSocketRequestMessage.parse(message);
-    chargePoints.get(conn.getRemoteSocketAddress()).processMessage(webSocketMessage);
+    LOGGER.info("sent message to "
+            + conn.getRemoteSocketAddress()
```

Comment:
conn.getRemoteSocketAddress() in a variable ?

Reply:
Yes, I was tired when writing this code, will do

Reply:
Done